apps/wing-console/console/ui/src/ui/api-interaction.tsx

```diff
@@ -316,28 +316,10 @@ export const ApiInteraction = memo(
       ],
     );
 
-    const loadRoutesFromOpenApi = useCallback(() => {
```

Comment:
it handled the use case when the route was removed from the code while it is selected in the UI.
after your change, the routes are getting updated but the routes string still remains visible

Reply:
The route is being removed from the dropdown, but I wouldn't remove it from the input. 

For example, if the user wrote a custom route `/route-that-never-existed`, we would clean it up as well.